src/Notifier.js

```diff
@@ -192,12 +218,23 @@ var Notifier = {
         return enabled === 'true';
     },
 
-    setToolbarHidden: function(hidden) {
+    setToolbarHidden: function(hidden, persistent = true) {
```

Comment:
just fyi: you may know this already but default parameters are a feature of ES6, which means that we can't use them in some browsers. But it's ok to use in react-sdk because babel translates it for us.